ghcide/src/Development/IDE/LSP/Notifications.hs

```diff
@@ -133,7 +146,7 @@ descriptor recorder plId = (defaultPluginDescriptor plId desc) { pluginNotificat
       let globs = watchedGlobs opts
       success <- registerFileWatches globs
       unless success $
-        liftIO $ logDebug (ideLogger ide) "Warning: Client does not support watched files. Falling back to OS polling"
+        liftIO $ logWith recorder Debug LogWarnNoWatchedFilesSupport
```

Comment:
Incidentally, this makes no sense. Logging text that begins "warning" at debug. Let's just make this a warn log.